addons/grenades/CfgAmmo.hpp

```diff
@@ -17,6 +17,18 @@ class CfgAmmo {
         intensity = 40000;
         flareSize = 12;
     };
+    class ACE_40mm_Flare_white: F_40mm_White {
+        intensity = 1250000; // vanilla: 10000
+        timeToLive = 45; // vanilla: 25, ace changes to 60
```

Comment:
>ace changes to 60

But it says 45?

Reply:
ace changes the upper base class to 60 from 25, I'll improve wording
```
    class FlareBase: FlareCore {
        timeToLive = 60;
```